config/devel/manager_image_patch.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: devworkspace-controller
+        image: docker.io/amisevsk/che-workspace-controller:dev
```

Comment:
Should this image be quay.io/devworkspace/devworkspace-controller:next?

Reply:
Ah these files accidentally got mangled while I was figuring out the Makefile, they should be full of env vars to be replaced. That must've made testing kind of tedious.

The purpose of the additional files in `config/devel` is that they're templates that we fill with `envsubst` before running kustomize, to get the same functionality with env vars we have in the current makefile (see the `deploy` rule).